websites/docs-smoke-test/src/content/views/beta.mdx

```diff
@@ -2,7 +2,7 @@
 title: Beta page
 beta: true
 showTimeToRead: true
-timeToRead: true
```

Comment:
I think this was a smoke test against authors confusing the two time to read frontmatters.

Reply:
Ah, got it. I noticed it because is creating a warning in the gatsby build about types. But I reverted it